scarlet/google-services.json

```diff
@@ -6,12 +6,6 @@
   },
   "client": [
     {
-      "client_info": {
```

Comment:
Now it won't work for both pro and lite versions. You should add generated `google-services.json` file, which contains two clients: pro and lite/default.